doc/nrf/device_guides/working_with_nrf/nrf70/developing/constrained.rst

```diff
@@ -9,6 +9,24 @@ Operating with a resource constrained host
 
 This guide provides recommendations and guidelines for using the nRF7002 as a companion chip on resource-constrained hosts such as the nRF5340 SoC.
 
+Stack combinations
+******************
+The below table indicates the supported stack combinations for the nRF7002 chip. This should help understand the various use cases that are supported by the nRF7002 chip.
```

Comment:
Same comments as above, regarding "stacks", and "nRF7002".
Then, I would add a bit more context here, to indicate exactly the purpose of this part of the guide. The table is not about the nRF70 Series - it is about the supporting hosts.

Suggestion
"The table below indicates the support of various classes of Wi-Fi networking applications (denoted by the corresponding protocol stack capabilities)  running on the different nRF host MCUs combined with the nRF70 Series companion IC. The table should help users identify and select the appropriate host MCU depending on the IoT use-case
of interest."